Description: Drop harmful sed search/replace calls that cause the package to
 FTBFS since libxml has been multiarch'ified.
Author: Felix Geyer <debfx-pkg@fobos.de>
Bug: https://savannah.gnu.org/bugs/index.php?34835

--- a/config/pathtls.m4
+++ b/config/pathtls.m4
@@ -71,8 +71,6 @@
       sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\2/'`
     tls_config_micro_version=`$TLS_CONFIG $tls_config_args --version | \
       sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\3/'`
-    # Strip '-L/usr/lib' off since this is always in the link path.
-    TLS_LIBS=`echo $TLS_LIBS | sed -e 's|-L/usr/lib||'`
 
     if test "x$enable_tlstest" = "xyes" ; then
       ac_save_CFLAGS="$CFLAGS"
--- a/config/pathxml.m4
+++ b/config/pathxml.m4
@@ -108,8 +108,6 @@
       sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\2/'`
     xml_config_micro_version=`$XML_CONFIG $xml_config_args --version | \
       sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\3/'`
-    # Strip '-L/usr/lib' off since this is always in the link path.
-    XML_LIBS=`echo $XML_LIBS | sed -e 's|-L/usr/lib||'`
 
     if test "x$enable_xmltest" = "xyes" ; then
       ac_save_CFLAGS="$CFLAGS"
